test/e2e_mc/fixture.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_mc
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"testing"
+
+	"antrea.io/theia/test/e2e"
+)
+
+func createDirectory(path string) error {
+	return os.Mkdir(path, 0700)
+}
+
+func (data *MCTestData) setupLogDirectoryForTest(testName string) error {
+	path := filepath.Join(testOptions.logsExportDir, testName)
+	// remove directory if it already exists. This ensures that we start with an empty
+	// directory
+	_ = os.RemoveAll(path)
```

Comment:
```suggestion
	os.RemoveAll(path)
```
Don't need to store if not using

Reply:
Fixed. Thanks.